README.md

```diff
@@ -2,159 +2,185 @@
 
+#### `await db.ready()`
+
+Waits until internal state is loaded.
+
+Use it once before reading `db.version` unless you called any of the other APIs.
+
+#### `await db.close()`
+
+Fully close this bee, including its core.
+
+#### `db.version`
+
+Number that indicates how many modifications were made, useful as a version identifier.
+
 #### `await db.put(key, [value], [options])`
 
-Insert a new key. Value can be optional. If you are inserting a series of data atomically,
-or you just have a batch of inserts/deletions available using a batch can be much faster
-than simply using a series of puts/dels on the db.
+Insert a new key. Value can be optional.
 
-Options include:
+If you're inserting a series of data atomically or want more performance then check the `db.batch` API.
 
+`options` includes:
 ```js
 {
   cas (prev, next) { return true }
 }
 ```
 
 ##### Compare And Swap (cas)
-You have the option to pass a `cas` function as an option to `put` that controls whether the `put` succeeds. Given `bee.put(key, value, { cas })`, `cas` is passed the current node (i.e. `{ seq, key, value }`) in `bee` at `key` and the next _tentative_ node. Then `put` succeeds only if `cas` returns `true` and fails otherwise.
+`cas` option is a function comparator to control whether the `put` succeeds.
+
+By returning `true` it will insert the value, otherwise it won't.
+
+It receives two args: `prev` is the current node entry, and `next` is the potential new node.
 
 ```js
-const cas = (prev, next) => prev.value !== next.value
-const db = new Hyperbee(core, { keyEncoding: 'utf8', valueEncoding: 'utf8' })
-await db.put('key', 'value')
-console.log(await db.get('key')) // { seq: 1, key: 'key', value: 'value' }
-await db.put('key', 'value', { cas })
-console.log(await db.get('key')) // { seq: 1, key: 'key', value: 'value' }
-await db.put('key', 'value*', { cas })
-console.log(await db.get('key')) // { seq: 2, key: 'key', value: 'value*' }
+await db.put('number', '123', { cas })
+console.log(await db.get('number')) // => { seq: 1, key: 'number', value: '123' }
+
+await db.put('number', '123', { cas })
+console.log(await db.get('number')) // => { seq: 1, key: 'number', value: '123' }
+// Without cas this would have been { seq: 2, ... }, and the next { seq: 3 }
+
+await db.put('number', '456', { cas })
+console.log(await db.get('number')) // => { seq: 2, key: 'number', value: '456' }
+
+function cas (prev, next) {
+  // You can use same-data or same-object lib, depending on the value complexity
+  return prev.value !== next.value
+}
 ```
 
-#### `{ seq, key, value } = await db.get(key)`
+#### `const { seq, key, value } = await db.get(key)`
 
-Get a key, value. If the key does not exist, `null` is returned.
-`seq` is the hypercore version at which this key was inserted.
+Get a key's value. Returns `null` if key doesn't exists.
 
-#### `await db.del(key, [options])`
+`seq` is the `db.version` (Hypercore length) at which this key was inserted.
```

Comment:
I think it should be correct to refer `seq` as the `db.version` there

Reply:
Just say hypercore index, no need to invent to many new concepts